main/store/migrate/migrations/39/index.ts

```diff
@@ -0,0 +1,18 @@
+import log from 'electron-log'
+import { v39StateSchema } from './schema'
+
+const migrate = (initial: unknown) => {
```

Comment:
I don't think we need a migration here, since we don't store the rates in the state, do we?